gen/build_deploy/bash.py

```diff
@@ -701,24 +707,26 @@ def do_create(tag, build_name, reproducible_artifact_path, commit, variant_argum
     """
     # TODO(cmaloney): Build installers in parallel.
     # Variants are sorted for stable ordering.
-    for variant, bootstrap_info in sorted(variant_arguments.items(),
-                                          key=lambda kv: pkgpanda.util.variant_str(kv[0])):
-        with logger.scope("Building installer for variant: ".format(pkgpanda.util.variant_name(variant))):
-            bootstrap_installer_name = '{}installer'.format(pkgpanda.util.variant_prefix(variant))
-            installer_filename = make_installer_docker(variant, all_completes[variant],
-                                                       all_completes[bootstrap_installer_name])
-
-            yield {
-                'channel_path': 'dcos_generate_config.{}sh'.format(pkgpanda.util.variant_prefix(variant)),
-                'local_path': installer_filename
-            }
+    for variant in sorted(variant_arguments.keys(), key=lambda k: pkgpanda.util.variant_str(k)):
+        variant_name = pkgpanda.util.variant_name(variant)
+        bootstrap_installer_name = '{}installer'.format(pkgpanda.util.variant_prefix(variant))
```

Comment:
`TODO`:

Everywhere  we are doing a string interpolation for the path, I think, defining template upfront will be a  preferable pattern. 

I am thinking something like this

```
dcos_launch_path= "{dcos_path_prefix}/dcos-launch"
dcos_launch_variant_path= "{dcos_path_prefix}/dcos-launch-{variant_info}"
bootstrap_name_template = "{pkgpanda_variant_path}/installer"
bootstrap_util_name_template = "{pkgpanda_variant_path}/util"
```

I'd define them all upfront, so that these are paths are like "routes are templatized" (as in webapps) and the code just substitutes the values at appropriate time.  Currently, everything is created when needed.